test/unit/local/chokidar_watcher.js

```diff
@@ -12,167 +12,167 @@ const metadata = require('../../../core/metadata')
 const Builders = require('../../support/builders')
 const configHelpers = require('../../support/helpers/config')
 const { ContextDir } = require('../../support/helpers/context_dir')
-const { onPlatforms } = require('../../support/helpers/platform')
+const { onPlatform } = require('../../support/helpers/platform')
 const pouchHelpers = require('../../support/helpers/pouch')
 
 const { TRAVIS } = process.env
 const { platform } = process
 
-describe('LocalWatcher Tests', function () {
```

Comment:
- Change test description to `ChokidarWatcher`
- Smaller diff with `if` + early return
- Also it doesn't actually fix broken tests, so not necessary yet